sites/svelte-5-preview/src/routes/docs/content/03-appendix/01-faq.md

```diff
@@ -116,7 +116,7 @@ npm i --save-dev prettier-plugin-svelte prettier
 
 ### When is it coming out?
 
-When it's done. The goal is sometime in early 2024.
+When it's done. The goal is sometime in late 2024.
```

Comment:
We currently have some more specific estimates/goals than this, but I think I'll leave it to others whether we want to put them here.

Reply:
Let's just say '2024' — we're very much doing everything in our power (without compromising) to avoid 'late'

```suggestion
When it's done. The goal is sometime in 2024.
```